MANIFEST.in

```diff
@@ -0,0 +1,6 @@
+graft pybamm
+prune tests
```

Comment:
This configuration matches the one with our previous version (23.5). We have historically never included the `tests/` directory and any of the other Markdown files except those inside the `pybamm/` folder. We could, however, provide some more files in the SDist, such as CITATION.cff and other project-specific files

Reply:
The LICENSE file is included in source distributions by default. I added the CITATION.cff file